config/kubernetes/sig-contributor-experience/teams.yaml

```diff
@@ -109,38 +69,40 @@ teams:
     - thockin
     - xiang90
     privacy: closed
-  sig-contributor-experience-pr-reviews:
-    description: PR reviews for contributor-experience infrastructure, such as the
-      PR bot, etc.
-    maintainers:
-    - cblecker
-    - idvoretskyi
-    - nikhita
-    members:
-    - grodrigues3
-    - idealhack
-    - rmmh
-    privacy: closed
-  sig-contributor-experience-proposals:
-    description: Design proposals for new contributor-experience infrastructure, such
-      as bots, monitoring dashboards, portals, etc.
-    maintainers:
-    - cblecker
-    - nikhita
-    members:
-    - grodrigues3
-    - idealhack
-    privacy: closed
-  sig-contributor-experience-test-failures:
-    description: List to triage test failures for contributor-experience infrastructure,
-      such as the PR bot, etc.
-    maintainers:
-    - cblecker
-    - nikhita
-    members:
-    - grodrigues3
-    - idealhack
-    privacy: closed
+    previously:
+      - sig-contributor-experience-misc-use-only-as-a-last-resort
+    teams:
+      sig-contributor-experience-apac-coordinators:
+        description: Contributor Experience APAC Coordinator team
+        maintainers:
+        - nikhita
+        members:
+        - alisondy
+        - aravindputrevu
+        - idealhack
+        - Pensu
+        - saiyam1814
+        - tao12345666333
+        privacy: closed
+      sig-contributor-experience-leads:
+        description: Chairs and Technical Leads for SIG Contributor Experience
+        maintainers:
+        - cblecker
+        - nikhita
+        members:
+        - parispittman
+        - Phillels
+        privacy: closed
+      sig-contributor-experience-pr-reviews:
+        description: PR reviews for contributor-experience infrastructure, such as the
+          PR bot, etc.
+        maintainers:
+        - cblecker
+        - idvoretskyi
+        - nikhita
+        members:
+        - idealhack
+        privacy: closed
   youtube-admins:
```

Comment:
nzoueidi is a new YouTube admin and needs to be added here.

Reply:
We've added (as of today) @markyjackson-taulia and @onlydole as well. I wouldn't consider it a blocker to getting this merged though. There will likely be some other updates coming in the youtube/zoom space soon.

Reply:
Agreed! We can cut additional PRs for those additions 👍